azext_edge/tests/edge/orchestration/resources/test_dataflow_endpoints_unit.py

```diff
@@ -113,3 +124,82 @@ def test_dataflow_endpoint_list(mocked_cmd, mocked_responses: responses, records
 
     assert result == mock_dataflow_endpoint_records["value"]
     assert len(mocked_responses.calls) == 1
+
+
+def test_dataflow_endpoint_delete(mocked_cmd, mocked_responses: responses):
+    dataflow_endpoint_name = generate_random_string()
+    instance_name = generate_random_string()
+    resource_group_name = generate_random_string()
+
+    mocked_responses.add(
+        method=responses.DELETE,
+        url=get_dataflow_endpoint_endpoint(
+            dataflow_endpoint_name=dataflow_endpoint_name,
+            resource_group_name=resource_group_name,
+            instance_name=instance_name,
+        ),
+        status=204,
+    )
+    delete_dataflow_endpoint(
+        cmd=mocked_cmd,
+        endpoint_name=dataflow_endpoint_name,
+        instance_name=instance_name,
+        resource_group_name=resource_group_name,
+        confirm_yes=True,
+        wait_sec=0.25,
+    )
+    assert len(mocked_responses.calls) == 1
+
+
+@pytest.mark.parametrize(
+    "scenario",
+    [
+        {"file_payload": {generate_random_string(): generate_random_string()}},
```

Comment:
should there be any checks for minimum properties in the file? A user can pass in an empty file too - does the service/cli handle that gracefully?

Reply:
yeah service will return a schema validation error quickly due to lacking of required resource

Reply:
talked offline